swarm_copy/tools/bluenaas_memodel_getall.py

```diff
@@ -29,9 +29,11 @@ class InputMEModelGetAll(BaseModel):
     page_size: int = Field(
         default=20, description="Number of results returned by the API."
     )
-    model_type: Literal["single-neuron-simulation", "synaptome-simulation"] = Field(
```

Comment:
As in other MRs, the name is not fitting.

Reply:
I already fixed this in the other MR:
https://github.com/BlueBrain/neuroagent/pull/44
When that PR is merged I will merge that onto this PR.